test/integration/confirmations/signatures/personalSign.test.tsx

```diff
@@ -0,0 +1,259 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import { ApprovalType } from '@metamask/controller-utils';
+import mockMetaMaskState from '../../data/integration-init-state.json';
+import { integrationTestRender } from '../../../lib/render-helpers';
+import { shortenAddress } from '../../../../ui/helpers/utils/util';
+import * as backgroundConnection from '../../../../ui/store/background-connection';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+  MetaMetricsEventLocation,
+} from '../../../../shared/constants/metametrics';
+
+jest.mock('../../../../ui/store/background-connection', () => ({
+  ...jest.requireActual('../../../../ui/store/background-connection'),
+  submitRequestToBackground: jest.fn(),
+}));
+
+const mockedBakcgroundConnection = backgroundConnection as jest.Mocked<
+  typeof backgroundConnection
+>;
+
+const backgroundConnectionMocked = {
+  onNotification: jest.fn(),
+};
+
+describe('PersonalSign', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
```

Comment:
My understanding was that `resetAllMocks` functionally includes `clearAllMocks`.